source/guide_teamspeak.rst

```diff
@@ -0,0 +1,286 @@
+
+.. code-block:: console
+
+  [isabell@stardust ts3server]$ ./ts3server_minimal_runscript.sh inifile=ts3server.ini
+  2020-02-14 21:43:43.659472|INFO    |ServerLibPriv |   |TeamSpeak 3 Server 3.11.0 (2020-01-13 08:12:37)
+  [...]
+  2020-02-14 22:17:31.884829|INFO    |              |   |Puzzle precompute time: 703
+  2020-02-14 22:17:31.885977|INFO    |FileManager   |   |listening on 0.0.0.0:47304, [::]:47304
+  2020-02-14 22:17:31.895316|INFO    |VirtualServerBase|1  |listening on 
+  2020-02-14 22:17:31.896826|INFO    |Query         |   |listening for query on 0.0.0.0:47305, [::]:47305
+  2020-02-14 22:17:31.897214|INFO    |Query         |   |listening for query ssh on 0.0.0.0:47306, [::]:47306
+  2020-02-14 22:17:31.897479|INFO    |CIDRManager   |   |updated query_ip_whitelist ips: 127.0.0.1/32, ::1/128,
+
+.. note:: If this does not work right now, it might be, that the ports are not yet open. Try again in a couple minutes. 
+
+If you want, you can also try to connect to the teamspeak server with your local teamspeak client software. Please use the IP address of your server the the default_voice_port (<IPofYourServer>:<default_voice_port>)
+
+Example:
+
+  .. image:: _static/images/teamspeak_client_connect.png
+      :align: center
+
+
+Configure the supervisord service
+=================================
+
+Since teamspeak does come with a start-stop script, all we need to do is to create the ``supervisor`` ini-file.
+
+Create supervisord ini (e.g. ``/home/$USER/etc/services.d/ts3server-daemon.ini``):
+
+::
+  
+  [program:ts3server]
+  directory=%(ENV_HOME)s/ts3server/
+  command=%(ENV_HOME)s/ts3server/ts3server inifile=ts3server.ini
+  startsecs=10
+  autostart=yes
+  autorestart=yes
+
+After creating the configuration, tell supervisord to refresh its configuration and start the service:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ supervisorctl reread
+  ts3server: available
+  [isabell@stardust ~]$ supervisorctl update
+  ts3server: added process group
+  [isabell@stardust ~]$ supervisorctl status
+  ts3server                        RUNNING   pid 4447, uptime 0:00:13
+  [isabell@stardust ~]$
+
+If it’s not in state RUNNING after 10 seconds, check your configuration and the log file in ``$HOME/logs/supervisord.log``.
+
+You can read more about the **supervisord** in the :manual:`supervisord <daemons-supervisord>` manual.
+
+
+Additional Notes
+================
+
+Gaining Admin Rights
+--------------------
+
+When you connect to your teamspeak server for the first time, you will need the token that got generated on first start of the server. If you have not taken a copy of the *token* and/or the *admin/password* combination, then you can get them from the log files located in the ``logs`` folder of your teamspeak installation. 
+
+Start and Stopping the Server
```

Comment:
This is the same for all services and documented in the manual. Please delete the section, so we don't have to maintain it at multiple places.

Reply:
deleted and used a more generic text with a link to the manual

Reply:
thanks!